install/nginx

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+set -e
+
+if [[ $# != 2 ]]
+then
+		echo "Usage: "
+		echo "	$0 userName fullURLToSite"
+		exit 1
+fi
+
+userName=$1
+FQDN=$2
+configFile=/etc/nginx/sites-available/$FQDN
+
+scriptDir="$(dirname $0)"
+cd $scriptDir
+# Read config file with paths to WP-installs and usernames
+source /usr/local/default/etc/wp-mass-admin/wpma.conf
+if [[ -e /usr/local/etc/wp-mass-admin/wpma.conf ]]; then
+    source /usr/local/etc/wp-mass-admin/wpma.conf
+fi
+
+if [[ "$useHttps" == "yes" ]]
+then
+	./use-https $userName $FQDN
+else
+	./no-https $userName $FQDN
+fi
+
```

Comment:
It is possible to change the config before running. 

However I agree with you that for the future the command line should offer more options, however I feel more like doing that in a proper python starter (replace the wpma bash script starter with one written in Python). I feel like this together with the installer refractor is a really large scope and perhaps we can delay these features to antoher PR.